.github/workflows/build.yml

```diff
@@ -146,7 +146,7 @@ jobs:
           whl_file=$(find dist -name "*.whl" | sed 's|^dist/||') # Find the built wheel file, remove dist/ prefix
           echo "Found built wheel file: $whl_file"
           PYGEN_FILE_PATH=$whl_file \
-          PACKAGES="[\"pyodide-http\", \"pandas\", \"typing-extensions>=4.10.0\", \"http://localhost:3000/dist/$whl_file\"]" \
+          PACKAGES="[\"pyodide-http\", \"pandas\", \"typing-extensions>=4.14.0\", \"http://localhost:3000/dist/$whl_file\"]" \
```

Comment:
This is related to a recent release of `typing-extensions`: https://pypi.org/project/typing-extensions/#history.

It is unrelated to the rest of the PR. The streamlit and jupyter apps in fusion will do the same update: https://cognitedata.slack.com/archives/C050CEHKNB1/p1749287564852289